### PR TITLE
Terraform and Bootstrap K8s nodes using the "environment"

### PR DESCRIPTION
**Pull request by akshaymankar**
Fixes https://github.com/zinfra/backend-issues/issues/1754

### CODE REVIEW

**Comment by akshaymankar** [comment]
I couldn't run any ansible on the qa-demo-2 environment as I don't have access to it.

**Comment by akshaymankar** [comment]
This causes staging environment to create and delete an ssh key. The VMs don't get touched so maybe it is not a problem.

File: ansible/Makefile [hunk 1]
```diff
@@ -42,6 +42,15 @@ bootstrap: check-env
 		--private-key ${ENV_DIR}/operator-ssh.dec \
 		-vv
 
+.PHONY: kube-minio-static-files
+kube-minio-static-files: check-env
+	poetry run ansible-playbook ${ANSIBLE_DIR}/kube-minio-static-files.yml \
```
Comment (flokli) [inline ansible/Makefile:47]: Please add a link to https://github.com/zinfra/backend-issues/issues/1763 here.

File: ansible/kube-minio-static-files.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# WARNING: This is not recommended for production use.
```
Comment (flokli) [inline ansible/kube-minio-static-files.yml:1]: same: https://github.com/zinfra/backend-issues/issues/1763

File: terraform/modules/hetzner-kubernetes/main.tf [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+resource "hcloud_server" "node" {
+  count       = var.node_count
+  name        = "${ var.environment }-kubenode${ format("%02d", count.index + 1 )}"
+  image       = var.image
+  server_type = var.server_type
+  ssh_keys    = var.ssh_keys
+  location    = var.location
+
+  labels = {
+    # FUTUREWORK: This label name is very undecriptive and it should be renamed
```
Comment (flokli) [inline terraform/modules/hetzner-kubernetes/main.tf:12]: Followup ticket and link to it here?

Comment (akshaymankar) [reply terraform/modules/hetzner-kubernetes/main.tf:12]: I don't think this kind of refactorings need a ticket. We have such FUTUREWORK comments in a quite a few places. They are useful to someone already making a breaking change to also do it in that place.

**Review by lucendio — APPROVED** [review]

**Pull request merged**